### DISCUSSION TITLE
Build release for 1.19 Experimental snapshot 1?

### DISCUSSION THREAD

**Discussion by Jack7860** [IrisShaders/Iris#1302] [Q&A] [answered]
Just wondering if anyone would have the time to make a build for the new snapshot as i am pretty sure you did make build for the old 1.18 snapshots.

**Comment by Lolothepro** [comment]
It is available on Discord

**Comment by AlsoSylv** [comment, marked as answer]
There are builds for 1.19 in our discord, and the branch is on github, but no builds for 1.19 will be posted on CF or Modrinth until it releases officially, and not as an experimental snapshot
